docs/aws/aws.md

```diff
@@ -100,23 +101,29 @@ repeat the verification process for each subnet ID.
    for the same region where your subnet is.
 
    5. Execute:
-    
+
        ```shell        
-       ./osd-network-verifier egress \
-           --subnet-id <subnet_id> 
+      # using AWS profile 
+      ./osd-network-verifier egress --subnet-id $SUBNET_ID --profile $AWS_PROFILE
+      
+      # using AWS secret 
+        AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY  \
+      ./osd-network-verifier egress --subnet-id $SUBNET_ID  
         ```
    
         Additional optional flags for overriding defaults:
       ```shell
-      --cloud-tags stringToString   (optional) comma-seperated list of tags to assign to cloud resources (default [osd-network-verifier=owned,red-hat-managed=true,Name=osd-network-verifier])
+      --cloud-tags stringToString   (optional) comma-seperated list of tags to assign to cloud resources e.g. --cloud-tags key1=value1,key2=value2 (default [osd-network-verifier=owned,red-hat-managed=true,Name=osd-network-verifier])
       --debug                       (optional) if true, enable additional debug-level logging
-      --image-id string             (optional) specify cloud image for the compute instance
-      --instance-type string        compute instance type (default "(optional) t3.micro")
+      --image-id string             (optional) cloud image for the compute instance
+      --instance-type string        (optional) compute instance type (default "t3.micro")
       --kms-key-id string           (optional) ID of KMS key used to encrypt root volumes of compute instances. Defaults to cloud account default key
-      --region string               (optional) compute instance region. If absent, environment var AWS_DEFAULT_REGION will be used, if set (default "us-east-1")
+      --region string               (optional) compute instance region. If absent, environment var AWS_REGION will be used, if set (default "us-east-2")
+      --profile string              (optional) AWS profile. If present, any credentials passed with CLI will be ignored.
```

Comment:
This doesn't work if we don't specify the profile explicitly thru the flag, it would be nice if it was able to be grabbed from env as well:

```shell
export AWS_PROFILE=admintestaccount

env | grep AWS
AWS_PROFILE=admintestaccount

./osd-network-verifier egress --subnet-id subnet-0ed916fb8a22b3342 --region us-east-1
Using region: us-east-1
Unable to create AWS client: Instance type t3.micro is invalid: Unable to gather list of supported instance types from EC2: operation error EC2: DescribeInstanceTypes, failed to sign request: failed to retrieve credentials: static credentials are empty


./osd-network-verifier egress --subnet-id subnet-0ed916fb8a22b3342 --region us-east-1 --profile admintestaccount
Using region: us-east-1
Using AWS profile: admintestaccount
Created instance with ID: i-02955036ff1cd1d8d
EC2 Instance: i-02955036ff1cd1d8d Running
Gathering and parsing console log output...
Terminating ec2 instance with id i-02955036ff1cd1d8d
.
.
```

Reply:
Agreed. Not sure if that's in the scope here, but we need it. We also need it for AWS key and secret.. those envs aren't working either. I'll make a card to read keys and profile from env vars. I think we should leave this one limited to the flag param.

Reply:
Jira card: https://issues.redhat.com/browse/OSD-12432